### PR TITLE
7584.master

### PR DESCRIPTION
**Pull request by ivandudardb**
Added and utilized the AddToAny module.
Arranged blocks on the page using Layout Builder.
Revised the user block (made it using Views).

### CODE REVIEW

**Review by HeorhiiBoliuk — APPROVED** [review]
Looks good

File: web/themes/custom/customtheme/customtheme.theme [hunk 1]
```diff
@@ -34,3 +34,14 @@ function customtheme_preprocess_page(&$variables)
...
 }
+
+
+use Drupal\entity_pager\EntityPagerLinkInterface;
+
+/**
+ * Implements hook_entity_pager_link_alter().
+ */
+function mymodule_entity_pager_link_alter(EntityPagerLinkInterface $link) {
+  // Модифікуємо значення #title для посилання.
```
Comment (lavrenchuk-dev) [inline web/themes/custom/customtheme/customtheme.theme:45]: ごめんなさい、ウクライナ語はわかりません

**Review by lavrenchuk-dev — CHANGES_REQUESTED** [review]

File: web/themes/custom/customtheme/customtheme.theme [hunk 2]
```diff
@@ -34,14 +34,3 @@ function customtheme_preprocess_page(&$variables)
...
-
-use Drupal\entity_pager\EntityPagerLinkInterface;
-
-/**
- * Implements hook_entity_pager_link_alter().
- */
-function mymodule_entity_pager_link_alter(EntityPagerLinkInterface $link) {
-  // Модифікуємо значення #title для посилання.
-  $link->text = 'New Title';
-}
```
Comment (ivandudardb) [inline web/themes/custom/customtheme/customtheme.theme]: These were my unsuccessful attempts to add a title to the pager

**Comment by lavrenchuk-dev** [comment]
@ivandudardb Commit message should describe changes that were made in that commit.
"Requested changes" doesn't describe anything, maybe in the context of PR, but just naming it "removed useless hook" is already much more descriptive.

**Pull request merged**